internal/app/referenceclient/raw_request_test.go

```diff
@@ -54,6 +54,26 @@ func TestRawRequestSender(t *testing.T) {
 	err = anypb.MarshalFrom(msgPayload, val, proto.MarshalOptions{})
 	require.NoError(t, err)
 
+	var requests sync.Map // map[string]chan *http.Request
```

Comment:
The changes in this file are admittedly unrelated. I was just moving stuff around to make it a little more consistent with the other tests (and maybe a little more readable?).